### ISSUE TITLE
Strange namespace errors after adding package to project 

### ISSUE THREAD

**Issue by DaReign** [google/play-unity-plugins#166]
I added to project com.google.play.assetdelivery-1.6.0.unitypackage

And errors appears after I tried to build project
![image](https://user-images.githubusercontent.com/11806754/147151633-86aaff1d-e280-46a7-b27b-44cc14f696e7.png)
My unity version is 2019.2.20f1

Adding this package shouldn't brake project in such way. It looks that it blocks compiling some libraries but I completly don't understand it.

**Comment by jamcohen** [comment]
Hi @DaReign,

Thanks for bringing this up. I'm not sure what would cause this issue.

I failed to reproduce the issue after going through these steps:
1. Create a new Unity project in 2019.2.20f1.
2. Switch to Android platform.
3. Import the com.google.play.assetdelivery-1.6.0.unitypackage.
4. Build the app by clicking Build in the "File -> Build settings" window.
5. Build the app by selecting "Google -> Build Android App Bundle" from the menu bar.

A couple questions to help me narrow down the issue:
1. Do the errors only appear when you build, or do they appear when you open the editor?
2. How are you building the app?
3. Have you tried deleting your Library and Temp folders and re-opening the project? Does that fix the issue?
4. What are your Android Resolver settings? You can find these by selecting "Assets -> External Dependency Manager -> Android Resolver -> Settings" in the menu bar. See below for the settings I used to test.
<img width="430" alt="Screen Shot 2021-12-22 at 1 55 40 PM" src="https://user-images.githubusercontent.com/4805315/147160140-7a7ee5f0-a0d8-4c00-b94e-90568f5589a5.png">

**Comment by DaReign** [comment]
ad 1. After first aproach to build by clicking Build in the "File -> Build settings" window errors show up. "Google" option in menu dissapeared.

ad.2  By clicking Build in the "File -> Build settings" window. I can't go to next step  by selecting "Google -> Build Android App Bundle" from the menu bar because after first operation errors showed up and "Google" menu dissapeared.

ad.3 I tried this as you suggested but I didn't helped. I tried also reimport assets but errors are still there.

ad.4
![image](https://user-images.githubusercontent.com/11806754/147281435-a34bc19d-1e09-40dd-ac68-0485fa41eadb.png)

**Comment by jamcohen** [comment]
I think the issue has something to do with the External Dependency Manager (EDM4U). Your Android Resolver Settings windows says that your Android Resolver version is 1.2.130, which is different from the one we include in the .unitypackage (1.2.167).

My guess is that your project contained version 1.2.130 of EDM4U and when you installed version 1.2.167 in com.google.play.assetdelivery-1.6.0.unitypackage, the old files and the new files conflicted in some way. Newer versions of EDM4U are designed to handle this case but 1.2.130 might not be. 

I would try deleting the folder under Assets/ExternalDependencyManager and reimporting the com.google.play.assetdelivery-1.6.0.unitypackage. Then delete the Library folder and reimport just to be safe.

**Comment by DaReign** [comment]
Should I delete whole Library folder with project settings, addresssables config files etc ?

**Comment by jamcohen** [comment]
Yes. Feel free to keep a backup in case you lose important data, but generally everything in the Library folder can be generated from other files (e.g. Asset's .meta files, the files within the ProjectSettings folder). This happens when Unity opens and imports Assets. 

However, I haven't used addressables so I don't know if those config files are reconstructed.

**Comment by DaReign** [comment]
Well I asked this because my library folder have 20GB :) :) :) I know that my project is huge ;). And last time regenerating took some time :) So I just want to know what files I can leave because I don't want to setup again whole project. Deleting library folder switch for example project from android to PC , and I have to setup this again from scratch and because project is for VR there are also additional other steps :(. Maybe there is easier approach. You mentioned that I have older versions of EDM4U, maybe I could download / install newer version. Forgive me if it is stupid question :) but my level of knowlegde of this stuff is less than basic .

**Comment by jamcohen** [comment]
Ah I see. In that case we can wait to delete the Library folder as a last resort.

Did you try deleting the Assets/ExternalDependencyManager folder and reimporting the com.google.play.assetdelivery-1.6.0.unitypackage? Once that's done, check the top of the Asset Resolver Settings for the version number. It should be 1.2.167.

**Comment by DaReign** [comment]
Yes I did this
![image](https://user-images.githubusercontent.com/11806754/147415418-6eb60f52-90e9-4fe3-9a8e-2d622cfd28ca.png)

But the errors still exist.
